test/alphabet/aminoacid_test.cpp

```diff
@@ -36,12 +36,15 @@
 
 #include <gtest/gtest.h>
 
+#include <range/v3/view/concat.hpp>
+#include <range/v3/view/single.hpp>
 #include <range/v3/view/zip.hpp>
 
 #include <seqan3/alphabet/concept.hpp>
 #include <seqan3/alphabet/aminoacid/all.hpp>
 
 using namespace seqan3;
+using namespace seqan3::detail;
```

Comment:
You should not import the detail namespace, instead use `detail::`. Otherwise you might not find namespace mistakes in your code.